### PR TITLE
unrpa: init at 2.3.0

### PR DESCRIPTION
**Pull request by leo60228**
###### Motivation for this change
I wanted to use it.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] N/A: Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] N/A: Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] N/A: Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/unrpa/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yl4qdwp3in170ks98qnldqz3r2iyzil5g1775ccg98qkh95s724";
+  };
+
+  propagatedBuildInputs = [ uncompyle6 ];
+
+  meta = with lib; {
+    homepage = https://github.com/Lattyware/unrpa;
```
Comment (jonringer) [inline pkgs/development/python-modules/unrpa/default.nix:17]: rfc 45
```suggestion
    homepage = "https://github.com/Lattyware/unrpa";
```

**Review by jonringer — COMMENTED** [review]
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

**Review by jonringer — COMMENTED** [review]
Also, commit message should be the `install-path: message`, so:
```
pythonPackages.unrpa: init at 2.3.0
```

File: pkgs/development/python-modules/unrpa/default.nix (hunk 1 above)
Comment (leo60228) [reply pkgs/development/python-modules/unrpa/default.nix:17]: The Python example in the Nixpkgs manual uses the old notation, fixing this.

**Comment by leo60228** [comment]
> Also, commit message should be the `install-path: message`, so:

Is this documented anywhere? [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) says it should be `pkg-name`.

**Comment by jonringer** [comment]
> Is this documented anywhere? [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) says it should be `pkg-name`.

If it was a top-level attribute, these two things are the same. However, this exists on the python package set, which is slightly different.

I agree that the wording isn't super explicit about what this means

**Comment by jonringer** [comment]
Reading about the package, it seems that we are more interested in the executable, rather than being able to import it as a library. So we should be using `buildPythonApplication`, please read https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonapplication-function for more details on packaging python applications with nix :)

**Comment by leo60228** [comment]
As using it as a library is a supported usecase, should `toPythonApplication` not be used instead? This is similar to youtube-dl (where usage as an application is much more common), which is the example for `toPythonApplication`.

**Comment by jonringer** [comment]
yea, that seems good

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 87677 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary>  - unrpa (python37Packages.unrpa)  - python38Packages.unrpa</details>

**Pull request merged**